samples/csharp/MicrosoftGraphSDK/Team.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.Graph;
+
+namespace MicrosoftGraphSDK
+
+
+{
+    public class Team
+    {
+        public static async Task<IUserJoinedTeamsCollectionPage> GetJoinedTeams(
```

Comment:
can be an extension method

Reply:
Thanks for your comment, changes have been added to the latest commit